changelog/unreleased/pr-21376.toml

```diff
@@ -0,0 +1,5 @@
+type = "fixed"
+message = "Fix when in Event definition (dark mode) error message is hard to read"
```

Comment:
The message is a bit hard to read, what about "Improve readability of filter preview error message in event definition wizard (dark mode)."?

Reply:
I would also prefer to not use a static color here.

I did not saw we are using `readableColor` here. I had a look at the function and we might be able to tweak it a bit to produce readable colors with a higher contrast.

This is how it looked like after tweaking the function, but I need to check further use cases before I can tell if the change makes sense.

![image](https://github.com/user-attachments/assets/47adadcf-1f5e-4c3d-9ea5-4525ded2925d)

![image](https://github.com/user-attachments/assets/d559577e-d279-455d-94e2-23b949aba7cd)